Automattic/WP-Job-Manager
# Add/usage tracking improvements

Fixes https://github.com/Automattic/WP-Job-Manager/issues/2580

### Changes proposed in this Pull Request

* Usage tracking checkbox in Setup is now checked by default
* The "Usage Tracking" notice will show up 1 week after the plugin has been activated
* When the plugin is updated, we'll show the "Usage Tracking" notice again, but only one time. If it's dismissed, it won't show up on future updates
* I also added a few `target="_blank"` attributes to some links that did not have them.

### Testing instructions

* Pull this branch to a test site
* Make sure you DON'T have usage tracking enabled
* Go to run a setup on this page:  `/wp-admin/index.php?page=job-manager-setup`
* See that the box is checked by default
* Click `Save and Skip setup` and make sure the option is saved properly
---
* To mimic the 1 week wait, you can edit the time in `delay_tracking_notice` to display the notice.
* To mimic a plugin update, you can edit the `WP_Job_manager::updater()` function so that the condition is true and so that the code runs
* Make sure that the usage tracking prompt shows up after that.
